src/main/kotlin/bedwars/wool_color.kt

```diff
@@ -0,0 +1,19 @@
+package bedwars
+
```

Comment:
Any reason we're not using org.bukkit.DyeColor?

Reply:
ok sure

Reply:
actually it's less convenient and it's deprecated